demo/demo.md

```diff
@@ -381,6 +381,62 @@ Use the `slim` attribute along with `iconOnly` attribute on the `auro-button` el
 
 </auro-accordion>
 
+## Rounded
+
```

Comment:
Let's add some descriptive language. We should explain how this button is different and explain the slots here since we divided them up to achieve the animation.